runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -6235,6 +6241,29 @@ static int32_t J9THREAD_PROC samplerThreadProc(void * entryarg)
             // compute jit state
             jitStateLogic(jitConfig, compInfo, diffTime); // Update JIT state before going to sleep
 
+            // detect high code cache occupancy
+            TR::CodeCacheManager *manager = TR::CodeCacheManager::instance();
+            size_t usedCacheSize = manager->getCurrTotalUsedInBytes();
+            size_t threshold = manager->codeCacheConfig().highCodeCacheOccupancyThresholdInBytes();
+            if ((usedCacheSize > threshold) && !highCodeCacheOccupancyThresholdReached)
+               {
+               highCodeCacheOccupancyThresholdReached = true;
+               if (TR::Options::getCmdLineOptions()->isAnyVerboseOptionSet(TR_VerbosePerformance))
+                  {
+                  TR_VerboseLog::writeLineLocked(TR_Vlog_PERF, "t=%6u JIT entered high code cache occupancy state",
+                     (uint32_t)persistentInfo->getStartTime());
```

Comment:
Here we don't want to print the start-up time but the elapsed time

Reply:
I believe this is referred as `crtTime` in this context

Reply:
Strange. I thought that the times were correct in Friday's tests.